Jenkinsfile

```diff
@@ -28,36 +26,37 @@ pipeline {
                         """
                     }
                  }
+                stage('Install deps') {
+                    steps {
+                        bat '''
+                            cd C:\\Users\\ContainerAdministrator\\ingenialink-python
+                            python -m venv venv
+                            venv\\Scripts\\python.exe -m pip install -r requirements\\dev-requirements.txt
+                            venv\\Scripts\\python.exe -m pip install -e .
+                        '''
+                    }
+                }
                 stage('Build wheels') {
                     steps {
-                        script {
-                            PYTHON_VERSIONS.each { version ->
-                                stage("Build Python ${version} wheel") {
-                                    bat """
-                                        cd C:\\Users\\ContainerAdministrator\\ingenialink-python
-                                        py -${version} -m venv py${version}
-                                        py${version}\\Scripts\\python.exe -m pip install -r requirements\\dev-requirements.txt
-                                        py${version}\\Scripts\\python.exe -m pip install -e .
-                                        py${version}\\Scripts\\python.exe setup.py build sdist bdist_wheel
-                                    """
-                                }
-                            }
-                        }
+                        bat '''
+                             cd C:\\Users\\ContainerAdministrator\\ingenialink-python
+                             venv\\Scripts\\python.exe setup.py build sdist bdist_wheel
+                        '''
                     }
                 }
                 stage('Check formatting') {
                     steps {
                         bat """
                             cd C:\\Users\\ContainerAdministrator\\ingenialink-python
-                            py${PYTHON_VERSIONS[-1]}\\Scripts\\python.exe -m black -l 100 --check ingenialink tests
+                            venv\\Scripts\\python.exe -m black -l 100 --check ingenialink tests
```

Comment:
I would add this flag `--experimental-string-processing` for string checking.
What do you think?

Reply:
Done